ckcp/openshift_dev_setup.sh

```diff
@@ -360,6 +381,9 @@ install_pipeline_service() {
     WEBHOOK_SECRET="placeholder_webhook" \
     "$GITOPS_DIR/pac/setup.sh" | indent 4
 
+  echo -n "- Install tekton-results DB: "
+  kubectl apply -k "manifests/tekton-results/openshift" >/dev/null 2>&1
```

Comment:
Do not redirect the output, instead use `indent` to format it properly.

Reply:
replaced with Indent